### PR TITLE
rename thumbup to like + add new icons

### PR DESCRIPTION
**Pull request by jonathanadler**

### CODE REVIEW

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-icons-common**
Commit: **5563b307307b63dee4b2e29ff2300ad32f0d0cc3**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-icons-common.pr_2249/

File: packages/wix-ui-icons-common/src/on-stage/general/metadata.ts [hunk 1]
```diff
@@ -268,12 +268,26 @@ const tpaIconsMetadata: Array<IconMetadata> = [
...
+    tags: [],
+    sizes: { "24":"Like" },
+  },
   {
     title: "ThumbUp",
     category: "General",
     description: "",
     tags: [],
-    sizes: { "24":"ThumbUp" },
+    sizes: { "24":"ThumbUpMedium" },
```
Comment (sivanharel) [inline packages/wix-ui-icons-common/src/on-stage/general/metadata.ts:283]: What it the conversions for the names of the icons?
Do we add a `Medium` suffix or just a `Small` and `Large`?
As I understood we don't add a `Medium` suffix for the medium size and just call it as the name of the icon 🤔

Comment (jonathanadler) [reply packages/wix-ui-icons-common/src/on-stage/general/metadata.ts:283]: You're right!
I'll change the file names

Comment (jonathanadler) [reply packages/wix-ui-icons-common/src/on-stage/general/metadata.ts]: I'll also make these changes to the `reactions` folder

**Review by sivanharel — APPROVED** [review]
💪

File: packages/wix-ui-icons-common/src/on-stage/general/metadata.ts (hunk 1 above)
Comment (sivanharel) [reply packages/wix-ui-icons-common/src/on-stage/general/metadata.ts]: cool :)

**Pull request merged**